editor/selectors.js

```diff
@@ -3,6 +3,7 @@
  */
 import moment from 'moment';
 import { first, last, get } from 'lodash';
+import { createSelector } from 'reselect';
```

Comment:
mmm looks like https://github.com/toomuchdesign/re-reselect do exactly what I've proposed.

But I think I'll just switch to `rememo`